controller/error_pages.rst

```diff
@@ -284,70 +284,12 @@ the request that will be dispatched to your controller. In addition, your contro
 will be passed two parameters:
 
 ``exception``
-    A :class:`\\Symfony\\Component\\ErrorHandler\\Exception\\FlattenException`
-    instance created from the exception being handled.
+    A :class:`Throwable` the original exception being handled.
```

Comment:
Why not keep `A Throwable instance created from the original...`?

Reply:
I've reworded the statement. wdyt? thanks.

side note: it isn't created from the original, but it's the original exception.